mkdocs.yml

```diff
@@ -39,3 +39,8 @@ theme:
 
 extra_css:
   - stylesheets/extra.css
+
+
+nav:
+  - 'README.md'
+  - 'openid-connect/openid-connect.md'
```

Comment:
Instead of adding nav, you would edit [handbook/.pages](https://github.com/ni/systemlink-operations-handbook/blob/master/handbook/.pages).